spec/models/organization_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'spec_helper'
+
+describe Organization do
+  let(:organization) { create(:organization) }
+
+  describe 'validation' do
+    it 'has a valid factory' do
```

Comment:
This test is not necessary, since we have https://github.com/openSUSE/osem/blob/master/lib/tasks/factory_girl.rake